hack/generate-yamls.sh

```diff
@@ -146,6 +150,7 @@ ${SERVING_CRD_YAML}
 ${SERVING_CERT_MANAGER_YAML}
 ${SERVING_ISTIO_YAML}
 ${SERVING_NSCERT_YAML}
+${SERVING_NAMESPACE_YAML}
```

Comment:
I'd get rid of this too and just do the `kubectl create namespace` in both cases below 🤷‍♂